tests/test_context.py

```diff
@@ -474,3 +474,8 @@ def test_get_all_raw_object_attributes(cursor):
     assert isinstance(raw_results, list)
     assert len(raw_results) > 0
     assert isinstance(raw_results[0], tuple)
+
+    # Make sure that this data is cached for future use
+    cursor.close()
+    raw_results_again = dbcontext.get_all_raw_object_attributes()
```

Comment:
the side-effect this is verifying is that it doesn't hit the db again?

Reply:
Exactly.